practice-app/website/test/events_search_query_test.py

```diff
@@ -0,0 +1,55 @@
+import unittest
+from ..api.event import query_handler_events
+
+class TestEventAPI(unittest.TestCase):
```

Comment:
Looks like we named our test class the same, maybe my error it is a general name but since my part is merged can you rename this class?

Reply:
done

Reply:
Great.